cmake/modules/autopas_eigen.cmake

```diff
@@ -1,4 +1,4 @@
-option(Eigen3_ForceBundled "Do not look for an installed version, always use bundled." OFF)
+option(Eigen3_ForceBundled "Do not look for an installed version, always use bundled." ON)
```

Comment:
is this intended?

Reply:
yes. I often ended up having to set those explicitly because the lib from another build folder was found.